hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java

```diff
@@ -335,8 +335,8 @@ public void execute() throws IOException {
 
       if (cmdline.hasOption(OPTION_YARN_QUEUE_NAME)) {
```

Comment:
I suggest moving this block to before the `super.execute();` that creates the connection. That way, you can directly update the conf using `getConf()`. Won't have any change in outcome, but feels a bit simpler.

Reply:
@DieterDP-ng  Thanks for your review. I have revised it. could you take a look? Thanks.

Reply:
Looks good, no further remarks from me.